src/app/app-routing.module.ts

```diff
@@ -1,11 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
+import { trimEnd } from 'lodash';
+
 import { LoginComponent } from './login/login.component';
 import { SigninComponent } from './signin/signin.component';
 import { ContextResolver } from './shared/context-resolver.service';
 
 
+export function removeAction(url: string) {
```

Comment:
Sounds good - like I say, I had to hack this, the right way is route the outlet as null, but it just doesn't work in the version of angular we are on.